install-paperless-ngx.sh

```diff
@@ -47,24 +47,29 @@ if [[ $(id -u) == "0" ]] ; then
 	exit 1
 fi
 
-if [[ -z $(which wget) ]] ; then
+if ! command -v wget &> /dev/null ; then
 	echo "wget executable not found. Is wget installed?"
 	exit 1
 fi
 
-if [[ -z $(which docker) ]] ; then
+if ! command -v docker &> /dev/null ; then
 	echo "docker executable not found. Is docker installed?"
 	exit 1
 fi
 
-if [[ -z $(which docker-compose) ]] ; then
-	echo "docker-compose executable not found. Is docker-compose installed?"
-	exit 1
+DOCKER_COMPOSE_CMD="docker-compose"
+if ! command -v ${DOCKER_COMPOSE_CMD} ; then
+	{
+		docker compose version &> /dev/null && DOCKER_COMPOSE_CMD="docker compose"
+		} || {
+		echo "docker-compose executable not found. Is docker-compose installed?"
+		exit 1
+	}
```

Comment:
This is a bit hard to read, I think. How about just using two "normal" `if`s?

```bash
if ! command -v ${DOCKER_COMPOSE_CMD} ; then
  if docker compose version &> /dev/null ; then
    DOCKER_COMPOSE_CMD="docker compose"
  else
    echo "docker-compose executable not found. Is docker-compose installed?"
    exit 1
  fi
fi
```

Reply:
I'm not really sure why I didn't go for that in the first place.  Updated now